specification/network/resource-manager/Microsoft.Network/preview/2024-03-01/webapplicationfirewall.json

```diff
@@ -783,6 +783,14 @@
           "items": {
             "$ref": "#/definitions/ManagedRuleGroupOverride"
           }
+        },
+        "computedDisabledRuleGroups": {
+          "readOnly": true,
+          "description": "Stores the computed list of disabled rule groups after consuming manifest",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ManagedRuleSetRuleGroup"
```

Comment:
Is there a reason we are not reusing the ManagedRuleGroupOverride definition and adding a new one which is essentially the same?

Reply:
The `ManagedRuleGroupOverride` stores overrides for rules, allowing each rule/rule group to be individually enabled or disabled. On the other hand, the newly added `ManagedRuleSetRuleGroup` is used to maintain a list of Rule Groups. Currently, it is being utilized to keep track of disabled rules or rule groups. And, it can be reused in the future to store any list of Rule Groups, regardless of their state.

Reply:
Update: refactored ManagedRuleSetRuleGroup, simplified `rules` array within `computedDisabledRuleGroups` from objects to strings